pkg/packagekit/assets/distribution.dist

```diff
@@ -3,7 +3,7 @@
 	<title>{{.Title}}</title>
 	<pkg-ref id="com.{{.Identifier}}.launcher">
 		<bundle-version>
-			<bundle CFBundleShortVersionString="{{.Version}}" CFBundleVersion="{{.Version}}" id="com.{{.Identifier}}.launcher" path="usr/local/kolide/Kolide.app"/>
+			<bundle CFBundleShortVersionString="{{.Version}}" CFBundleVersion="{{.Version}}" id="com.{{.Identifier}}.launcher" path="usr/local/{{.Identifier}}/Kolide.app"/>
```

Comment:
This does not appear to actually affect the output (it gets replaced anyway), but I thought it was better to not have incorrect data here in the first place